python/lsst/pipe/base/_quantumContext.py

```diff
@@ -371,7 +372,9 @@ def put(
                     " attributes must be passed as the values to put"
                 )
             for name, refs in dataset:
-                valuesAttribute = getattr(values, name)
+                valuesAttribute = getattr(values, name, None)
+                if valuesAttribute is None:
```

Comment:
Is the auto-warning complaining because the syntax here is `if thing is None` instead of `if not thing` ?

Reply:
What "auto-warning" are you talking about? I don't see one in Github CI below.

Reply:
It's now a walrus, too.

Reply:
I think the code coverage warning maybe the cause of confusion.

Reply:
AHA, yes, that was confusing me, I guess it's not a linter warning, but a test coverage warning. Weird that it always seemed to coincide with your usage of `if thing is None` 😆